driver.go

```diff
@@ -8,16 +8,26 @@ import (
 
 // register driver
 func init() {
-	sql.Register("maxcompute", &Driver{})
+	sql.Register("maxcompute", &ODPSDriver{})
 }
 
-// impls database/sql/driver.Driver
-type Driver struct{}
+// ODPSDriver impls database/sql/driver.Driver
+type ODPSDriver struct {
+	conn odpsConn
```

Comment:
I'm not sure if it is reasonable to maintain a `conn` inside the driver.

Reply:
Maybe it's okay, we need to maintain the `settings` struct in the DB object and the `Driver` struct can be exposed when we implement the `sql.database`.